docs/guides/core-concepts/retry-ability.mdx

```diff
@@ -29,25 +29,30 @@ fails, check out our guide on [Test Retries](/guides/guides/test-retries).
 
 ## Commands, Queries and Assertions
 
-While all methods you chain off of `cy` in your Cypress tests are commands,
-there are some different types of commands it's important to understand:
-**queries**, **assertions** and **actions** have special rules about
-retryability. For example, there are 4 queries, an action, and 2 assertions in
-the test below.
+While all methods you chain off of `cy` in your Cypress tests are commands, it's
+important to understand the different rules by which they operate. **Queries**
+link up, retrying the entire chain together, while **non-queries** only execute
+once. Assertions are a type of query that's specially displayed in the command
+log. For example, there are 7 queries (2 of them assertions) and 2 non-queries
+in the test below.
 
 ```javascript
 it('creates an item', () => {
+  // Non-query commands only execute once.
   cy.visit('/')
 
-  cy.focused() // query
-    .should('have.class', 'new-todo') // assertion
+  // The queries .focused() and .should() link together,
```

Comment:
```suggestion
  // The .focused() query and .should() assertion link together,
```

Reply:
.should() is a query, and I'm very intentionally treating it as one in the text. That's why I went through these guides in the first place, to make sure that point is built into our docs.